config/merge.go

```diff
@@ -54,6 +62,14 @@ func Merge(existingServices *ServiceConfigs, environmentLookup EnvironmentLookup
 
 	adjustValues(serviceConfigs)
 
+	if options.Postprocess != nil {
+		var err error
+		serviceConfigs, err = options.Postprocess(serviceConfigs)
```

Comment:
@joshwget do you have use cases for pre and post process ? Asking that to understand if passing it `serviceConfigs` is sufficient or passing more things (like the project or the config) in addition to the `serviceConfigs` would be better ?